tests/data/simpleFramerISD.json

```diff
@@ -1,96 +1,53 @@
 {
```

Comment:
I had not noticed, but this is one cause of the failing tests.  We need to maintain the simple ISD (that was removed). The more complex one is 'too' complex for the 'simle' testing being done.

Reply:
This should be:

```json
{
	"center_ephemeris_time": 50,
	"dt_ephemeris": 100,
	"focal2pixel_lines": [0.0, 0.0, 10],
	"focal2pixel_samples": [0.0, 10, 0.0],
	"focal_length_model": {
		"focal_length": 50,
		"focal_length_epsilon": 1.0
	},
	"image_lines": 15,
	"image_samples": 15,
	"interpolation_method": "lagrange",
	"number_of_ephemerides": 1,
	"optical_distortion": {
		"x": [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
		"y": [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
	},
	"radii": {
		"semimajor": 0.01,
        "semiminor": 0.01,
        "unit":"m"
	},
	"reference_height": {
		"maxheight": 1,
		"minheight": -1,
		"unit": "m"
	},
	"sensor_location": {
		"unit": "m",
		"x": 1000,
		"y": 0,
		"z": 0
	},
	"sensor_orientation": [0.0, -0.707106781186547, 0.0, 0.707106781186547],
	"sensor_velocity": {
		"unit": "m",
		"x": 0,
		"y": 0,
		"z": 0
	},
	"starting_detector_line": 0,
	"starting_detector_sample": 0,
	"starting_ephemeris_time": 0,
	"sun_position": {
		"x": 100,
		"y": 100,
		"z": 0
	},
	"sun_velocity": {
		"x": 0,
		"y": 0,
		"z": 0
	}
}

I need to checkout the plugin re: a comment from @jessemaple but this ISD should be good.
```